packages/lex-cli/src/types.ts

```diff
@@ -1,14 +1,16 @@
 export interface GeneratedFile {
   path: string
-  content: string
+  content?: string
 }
 
 export interface GeneratedAPI {
   files: GeneratedFile[]
 }
 
 export interface FileDiff {
-  act: 'add' | 'mod' | 'del'
+  act: 'add' | 'mod' | 'del' | 'leave'
   path: string
   content?: string
 }
+
+export type ModificationTimes = Record<string, number>
```

Comment:
I feel like a `Map<string, number>` are more accurate when working with key-value store. They are also "safer" than plain objects used as key value store (although, in this instance, file names are arguably unlikely to be named `toString` or `__proto__`).

the typing also better reflect missing entries: `map.get('bar')` is typed `V | undefined` whereas `({} as Record<string, number>).bar` will incorrectly by typed as as `number` (actual value is `undefined`)